react/features/speaker-stats/components/web/SpeakerStats.js

```diff
@@ -65,6 +65,16 @@ class SpeakerStats extends Component<Props> {
         this._onSearch = this._onSearch.bind(this);
     }
 
+    /**
+     * Resets the search criteria when component will unmount.
+     *
+     * @private
+     * @returns {void}
+     */
+    componentWillUnmount() {
+        this._onSearch();
```

Comment:
this is a bit weird. I think it would be clearer to either have an explicit reset function be called here, or explicitly pass an empty `criteria` (e.g. change `_onSearch` signature to accept an object instead)